docs/release-notes/december-2022.md

```diff
@@ -0,0 +1,66 @@
+- Improved responsiveness of search bars within Supply Chain > Vulnerabilities and Supply Chain > Advisories.
+
+## Semgrep App
+
+### Additions
+
+On the [Findings](https://semgrep.dev/orgs/-/findings/) page, you can now filter by rule category and rule confidence level.
+
+## Semgrep CLI
+
+These release notes include upgrades for versions ranging between 1.0.0 and 1.2.1.
+
+### Additions
+
+- JSON output: Added a `max_memory_bytes` field to the output of the  `semgrep --json --time` which corresponds to the amount of memory allocated during the OCaml phase of Semgrep. This is useful for telemetry purposes.
+- DeepSemgrep: If you have a Team tier account in Semgrep App, and you enable the DeepSemgrep setting, then `semgrep ci` automatically runs the DeepSemgrep engine instead of the regular Semgrep CLI engine on full scans (but not in PR scans). See the [DeepSemgrep](/deepsemgrep/deepsemgrep-introduction/) documentation for installation details.
+
+### Changes
+
+- Semgrep CLI does not print a summary of blocking rules unless it is invoked with `semgrep ci` subcommand. (Issue [#6651](https://github.com/returntocorp/semgrep/pull/6651))
+
+## Documentation updates
+
+### Additions
+
+- Added a new section to Semgrep App > Single sign-on (SSO) configuration to configure Semgrep with [Azure Active Directory](/semgrep-app/sso/#setting-up-saml-sso-with-azure-active-directory) (Azure AD).
+- Added a new document [Learning Semgrep App with a demo project](/semgrep-app/demo-project/).
+- Added section [Disabling rules](/semgrep-app/rule-board/#disabling-rules).
+- Added [Licensing document](/licensing/) which provides an overview of licenses used by different r2c products.
+
+### Changes
+
+- Updated [Getting started with Semgrep App](/semgrep-app/getting-started-with-semgrep-app/) to clarify how permissions are used by Semgrep, such as what files are read and what features are enabled by certain permissions.
+- Separated referential introductions from [Getting started with Semgrep Supply Chain](/semgrep-sc/scanning-open-source-dependencies/) into its a separate document, [Overview of Semgrep Supply Chain](/semgrep-sc/semgrep-supply-chain-overview/).
+- Updated [Installing DeepSemgrep](/deepsemgrep/deepsemgrep-introduction/#installing-deepsemgrep) section.
+- Updated [Filtering findings](/semgrep-app/findings/#filtering-findings) section with information about new filtering options.
+- The following documents have been moved out of the Experiments section as they are now considered GA:
+    - [Autofix](/writing-rules/autofix/)
+    - [Generic pattern matching](/writing-rules/generic-pattern-matching/)
+    - [Metavariable analysis](/writing-rules/metavariable-analysis/)
+    - Taint propagators - moved to [Taint tracking](/writing-rules/data-flow/taint-mode/#propagators) documentation
+- Updated screenshots in Semgrep App documentation, and many additional improvements, and fixes were made.
```

Comment:
```suggestion
- Updated screenshots in Semgrep App documentation. Many additional improvements and fixes were made.
```

Removed 2x 'and' oxford comma is only for 3 items or more (A,B, and C not A, and B)

Reply:
Thank you!